mantidproject/mantid
# Add absorption to powder reduction GUI

~Requires #29818 to be merged in first~

**Description of work.**

This adds the newly added option to SNSPowderReduction from #29818 to the Powder Diffraction Reduction GUI



**To test:**

Run the Powder Diffraction Reduction GUI and try the new Absorption options and make sure that SNSPowderReduction runs with the correct options. This can be done with File->Export

Check that the packing fraction calculation is correct, and the sample formula validator works.



Internal Ref [PD87](https://code.ornl.gov/sns-hfir-scse/diffraction/powder/powder-diffraction/-/issues/87)





---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/ReviewingAPullRequest.html)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there is GUI work does it follow the [GUI standards](http://developer.mantidproject.org/Standards/GUIStandards.html)?
- If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
